src/components/scene/vr-mode-ui.js

```diff
@@ -71,6 +45,8 @@ module.exports.Component = registerComponent('vr-mode-ui', {
 
     this.orientationModalEl = createOrientationModal(this.exitVR);
     this.el.appendChild(this.orientationModalEl);
+
+    this.attachEventListeners();
```

Comment:
I need to move the creation and and insertion before toggleOrientationModalIfNeeded gets called. I could listen for some event from when we insert in the `update` method. Why's it done in that method though? I'm curious.

Also, I've seen attachEventListeners methods in other prototypes. I know you try to minimize public API surface area, which is good.